lib/vsc/install/shared_setup.py

```diff
@@ -1608,6 +1631,13 @@ def action_target(self, target, setupfn=None, extra_sdist=None, urltemplate=None
         if do_cleanup:
             self.cleanup()
 
+        # add location of README to target long description
+        readme_file = self.locate_readme()
+        if 'long_description' in target:
+            log.info(('Ignoring the provided long_descripton.'
+                      'Please set it in the %s or disable vsc_description') % os.path.basename(readme_file))
```

Comment:
this has been a while since i read the code, but how is can use `vsc_description` to prevent the long_description being overwritten here?

Reply:
I indeed missed one check here on `vsc_description`. To avoid polluting the code with more conditionals, I have now used a slightly different approach by using a temporary attribute in `target` instead of `long_description`. This avoids any overwrites and the need of additional checks.